Makefile.base.mk

```diff
@@ -213,7 +213,7 @@ DGL_SYSTEM_LIBS  += -framework Cocoa
 endif
 
 ifeq ($(WINDOWS),true)
-DGL_SYSTEM_LIBS  += -lgdi32
+DGL_SYSTEM_LIBS  += -lgdi32 -lcomdlg32
```

Comment:
this should be optional, only added if file selector is not disabled

Reply:
There doesn't exist any variable yet, in the makefiles, which defines the file dialog to be enabled or not.
We can let --as-needed just do its job, and comdlg32 will not link.